presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaConnectorFactory.java

```diff
@@ -36,10 +37,19 @@
         implements ConnectorFactory
 {
     private final Optional<Supplier<Map<SchemaTableName, KafkaTopicDescription>>> tableDescriptionSupplier;
+    private final Module additionalModule;
```

Comment:
Maybe call this `consumerModule` since it's a required module (compared to other connectors where it's actually an optional, additional module)